docs/content/en/docs/configuration/keys.md

```diff
@@ -2852,10 +2852,11 @@ See also:
 
 ## WAF
 
-| Configuration key | Scope  | Default | Since |
-|-------------------|--------|---------|-------|
-| `waf`             | `Path` |         |       |
-| `waf-mode`        | `Path` | `deny`  | v0.9  |
+| Configuration key | Scope    | Default | Since |
+|-------------------|----------|---------|-------|
+| `waf`             | `Path`   |         |       |
+| `waf-mode`        | `Path`   | `deny`  | v0.9  |
+| `use-coraza`      | `Global` | `false` | v0.14 |
```

Comment:
This seems a bit misplaced here, leading to conflict with every new waf implementation. What about renaming to `modsecurity-use-coraza` and moving to the modsecurity group? A few words here talking about Coraza should be enough, e.g. something like "Currently the only supported value is `modsecurity`, which also supports Coraza endpoints.".